R/UK.R

```diff
@@ -495,6 +495,50 @@ UK <- R6::R6Class("UK",
           release_date = self$release_date
         )
       return(clean_data)
+    },
+
+    #' @description Run tests on UK class. Test data can be downloaded if
+    #' download = TRUE, or a requested snapshot file is not found, and store a
+    #' snap shot at the path provided. If an existing snapshot file is found
+    #' then just load this data to use in the next tests. Tests data can be
+    #' downloaded (if requested), cleaned, processed and returned.
+    #' @param self_copy R6class the object to test
+    #' @param download logical. To download the data (TRUE) or use a snapshot
+    #' (FALSE). Defaults to FALSE.
+    #' @param nhs_included_path character_array the path to save the downloaded
+    #' nhs data
+    #' snapshot to.
+    specific_tests = function(self_copy, download = FALSE,
+                              nhs_included_path = "") {
+      if (self_copy$level == "1") {
+        data_name <- "UK level 1 with 'nhsregions=TRUE'"
+        self_copy$nhsregions <- TRUE
+        source <- class(self_copy)[1]
+        if (!(nchar(nhs_included_path))) {
+          nhs_included_path <- paste0(
+            "custom_data/", source,
```

Comment:
as in previous review this won't work interactively if this folder doesn't exist.

Reply:
So I had kept it in so it could run from testthat but thinking about it I could remove the defaults from here and just have testthat pass in the file path